src/screens/ProfileScreen/index.js

```diff
@@ -96,12 +97,20 @@ const Header = (props) => {
         headerHeightRef.current = headerHeightLayout;
       }}>
       <View style={styles.content}>
-        <View style={{flexDirection: 'row'}}>
+        <View style={{flexDirection: 'row', alignContent: 'center', alignItems: 'center'}}>
           <ProfilePicture
             onImageContainerClick={changeImage}
             profilePicPath={dataMain.profile_pic_path}
+            karmaScore={Math.floor(dataMain.karma_score)}
           />
-          <View style={{marginLeft: 20}}>
+          <View
+            style={{
+              flexDirection: 'column',
+              paddingHorizontal: 14,
+              paddingVertical: 5,
+              justifyContent: 'center'
+            }}>
+            <KarmaScore score={Math.floor(dataMain.karma_score)} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `karmaScore` prop is being passed to the `ProfilePicture` component and the `KarmaScore` component. However, it's not clear if these components are expecting this prop or how they're using it. Please ensure that these components have been updated to handle this new prop appropriately.



<!-- This is an auto-generated comment by OSS CodeRabbit -->